src/foam/java/PropertyInfo.js

```diff
@@ -162,6 +164,13 @@ foam.CLASS({
             type: 'boolean',
             args: [ { name: 'o', type: 'Object' } ],
             body: `return ((${this.sourceCls.name}) o).${this.propName}IsSet_;`
+          },
+          {
+            name: 'isDefaultValue',
+            visibility: 'public',
+            type: 'boolean',
+            args: [ { name: 'o', type: 'Object' } ],
+            body: `return compareValues(get_(o), ${this.propValue}) == 0;`
```

Comment:
Would be worth adding a TODO that this needs to be revisiting when/if expression support is added to Java.